node/hubController.js

```diff
@@ -326,7 +327,7 @@ class HubController {
     }
 
     _createTranslator(translatorName, deviceInfo) {
-        this.ConsoleLogger.info(`_createTranslator ${translatorName}`);
+        this.logger.info(`_createTranslator ${translatorName}`);
         return this.OpenT2T.createTranslatorAsync(translatorName, deviceInfo).then( translator => {
```

Comment:
I'm thinking you should pass in the logger for a few reasons.

1.  We would have to take an array of logging transports in case we wanted to log to multiple places.
2.  The logger now handles telemetry as well using a set of telemetry transports so you would need to pass a second array in for that.

---
In reply to: [110027291](https://github.com/openT2T/cloud/pull/32#discussion_r110027291) [](ancestors = 110027291)